SmartDeviceLink/public/SDLChoiceSet.m

```diff
@@ -57,7 +58,9 @@ - (instancetype)initWithTitle:(NSString *)title delegate:(id<SDLChoiceSetDelegat
         return nil;
     }
 
-    if (timeout < 5 || timeout > 100) {
+    if (timeout == 0) {
+        SDLLogV(@"Creating a choice set with a 0 second timeout; the default timeout will be used: %f seconds", SDLChoiceSet.defaultTimeout);
+    } else if (timeout < 5 || timeout > 100) {
```

Comment:
This isn't capped this way in `AlertView`. We want the same system across both.

Reply:
Updated to cap the timeout range between 5 and 100 seconds.